terraform/loki/outputs.tf

```diff
@@ -27,6 +27,7 @@ output "endpoints" {
     loki_cluster                = "loki-cluster",
     receive_remote_write        = "receive-remote-write",
     self_metrics_endpoint       = "self-metrics-endpoint",
+    send_datasource             = "send-datasource",
```

Comment:
Do we also need an input for external relations?
I.e. would users of the high level cos tf module ever need to add additional datasources to the "circle of correlation"?